core/templates/dev/head/pages/exploration_player/tutor_card_directive.html

```diff
@@ -103,11 +102,11 @@
         <[interactionInstructions]>
         <i class="material-icons md-18" style="position: relative; top: 3px;">&#xE5C8;</i>
       </div>
-      <div ng-if="isViewportNarrow()">
+<!--       <div ng-if="isViewportNarrow()">
```

Comment:
?

Please either remove fully or uncomment. I suspect this is related to @kevinlee12's UI comment.

Reply:
done